ewatercycle/observation/grdc.py

```diff
@@ -1,92 +1,123 @@
 import os
-from datetime import datetime
+from pathlib import Path
+from typing import Dict, Tuple, Union
+import logging
 
 import pandas as pd
-
-
-def get_grdc_data(station_id,
-                  start_date,
-                  end_date,
-                  parameter='Q',
-                  data_home=None):
-    """
-    Get river discharge data from Global Runoff Data Centre (GRDC).
-
-    Requires the GRDC daily data files in a local directory.
-    The GRDC daily data files can be ordered at https://www.bafg.de/GRDC/EN/02_srvcs/21_tmsrs/riverdischarge_node.html
-
-    Parameters
-    ----------
-    station_id : str
-        The station id to get.
-        The station id can be found in the catalogues at https://www.bafg.de/GRDC/EN/02_srvcs/21_tmsrs/212_prjctlgs/project_catalogue_node.html
-    start_date : str
-        String for start date in the format: 'YYYY-MM-dd', e.g. '1980-01-01'
-    end_date : str
-        String for start date in the format: 'YYYY-MM-dd', e.g. '2018-12-31'
-    parameter : str, optional
-        The parameter code to get, e.g. ('Q') discharge, cubic meters per second
-    data_home : str, optional
-        The directory where the daily grdc data is located.
-        If left out will use the environment variable GRDC_DATA_HOME
-
-    Examples
-    --------
-    >>> from ewatercycle.observation.grdc import get_grdc_data
-    >>> data = get_grdc_data('6335020', '2000-01-01', '2001-01-01', data_home='.')
-    >>> data
-        <xarray.Dataset>
-        Dimensions:     (time: 367)
-        Coordinates:
-        * time        (time) datetime64[ns] 2000-01-01 2000-01-02 ... 2001-01-01
-        Data variables:
-            streamflow  (time) float64 ...
-        Attributes:
-            grdc_file_name:                6335020_Q_Day.Cmd
-            id_from_grdc:                  6335020
-            file_generation_date:          2019-03-27
-            river_name:                    RHINE RIVER
-            station_name:                  REES
-            country_code:                  DE
-            grdc_latitude_in_arc_degree:   51.756918
-            grdc_longitude_in_arc_degree:  6.395395
-            grdc_catchment_area_in_km2:    159300.0
-            altitude_masl:                 8.0
-            dataSetContent:                MEAN DAILY DISCHARGE (Q)
-            units:                         m�/s
-            time_series:                   1814-11 - 2016-12
-            no_of_years:                   203
-            last_update:                   2018-05-24
-            nrMeasurements:                NA
+from ewatercycle import CFG
+from ewatercycle.util import get_time
+
+logger = logging.getLogger(Path(__file__).name)
```

Comment:
Please use
```suggestion
logger = logging.getLogger(__name__)
```

so you get `ewatercycle.observation.grdc` as name instead of `grdc.py`. It is a bit more descriptive